Source/NSubstitute/Core/ICallResultsCache.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace NSubstitute.Core
+{
+    public interface ICallResultsCache
```

Comment:
I created a PR to address the ICallResultsCache. Please take a look. https://github.com/Zvirja/NSubstitute/pull/1/files